chain/consensus/filcns/upgrades.go

```diff
@@ -2998,7 +2999,21 @@ func (ml migrationLogger) Log(level rt.LogLevel, msg string, args ...interface{}
 	case rt.DEBUG:
 		log.Debugf(msg, args...)
 	case rt.INFO:
-		log.Infof(msg, args...)
+		if strings.Contains(msg, "jobs created") {
```

Comment:
this is a bit of a hack - how about you just go over and open a PR against go-state-types to change the log message to the one you're printing in this block